bin/kuzzle-createFirstAdmin.js

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env node
+var
+  readlineSync = require('readline-sync'),
+  clc = require('cli-color'),
+  request = require('request-promise'),
+  rc = require('rc'),
+  params = rc('kuzzle'),
+  name,
+  password,
+  step = 0,
+  // steps definitions
+  steps = [
+    {
+      // get username
+      prompt: '\n[❓] Which username for the first admin ?\nTip: avoid to name it admin, or root...\n: ',
+      check: function(userEntry) {
+        var _name = userEntry.trim(),
+        badNames = ['admin','root'];
+        if (badNames.indexOf(_name) === -1) {
+          if (_name.length < 4) {
+            console.log(clc.red('[✖] Username should have at least 4 chars'));
+            return false;
+          }
+
+          name = _name;
+          console.log(clc.green('[✔] Given username: ' + name));
+          if (_name !== userEntry) {
+            console.log(clc.cyan('[ℹ] The username has been trimmed.'));
+          }
+          name = _name;
```

Comment:
hmm 
either put "console.log(... + _name )" instead of "console.log(.... + name)" at line 26,
or move the "name = _name" before the console.log... ;)